src/style.css

```diff
@@ -1,7 +1,69 @@
+/* * {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    display: flex;
+} */
```

Comment:
- [ ] [OPTIONAL] Kindly remove all commented code to make your code base neater and easier to read :pray: